rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,206 @@
+# RFC 050: Design considerations for the concepts API
+
+This RFC collects some initial thinking on how we might represent concepts in the catalogue API.
+It's a starting point for discussions; not a final design.
+
+
+
+## Goals
+
+*   We want to be able to create per-person and per-subject pages on the website.
+    The data on these pages should come from the new concepts API.
+
+*   These pages should contain enough context to distinguish two people/subjects (e.g. whether "Charles Darwin" is the famous naturalist, his uncle, his son, or his grandson).
+
+*   We're not trying to replicate Wikidata -- the APIs shouldn't return large amounts of information we aren't going to use on wellcomecollection.org.
+
+*   Any new APIs should be similar to the existing works and images APIs.
+    We'll use the same conventions for filters, aggregations, including optional fields, and so on.
+
+
+
+## Endpoint #1: retrieve a single concept
+
+This is the single concept endpoint, which is used to render a page for an individual subject or person:
+
+```
+GET /concepts/azxzhnuh
+{
+  "id": "azxzhnuh",
+  "identifiers": [
+    {
+      "identifierType": "lc-names",
+      "value": "n12345678",
+      "type": "Identifier"
+    },
```

Comment:
Just noticed this isn't actually the `identifier` we use elsewhere: that looks like
```json
{
  "identifierType": {
    "id": "sierra-system-number",
    "label": "Sierra system number",
    "type": "IdentifierType"
  },
  "value": "b24367588",
  "type": "Identifier"
}
```